aws-storage-s3/src/androidTest/java/com/amplifyframework/storage/s3/AWSS3StorageUploadTest.java

```diff
@@ -195,6 +196,7 @@ public void testUploadFileIsCancelable() throws Exception {
      *         completed successfully before timeout
      */
     @Test
+    @Ignore("This test is failing reliably.")
```

Comment:
hmm is this an oxymoron? lol

Reply:
Haha yea. Actually, sorry, this `@Ignore` message is not setting a good example. It should have a bit more. Will update.

Reply:
haha I thought it was a funny message because "prone-to-failure" vs "reliability".
I think the reason for `@Ignore` is sufficient with this already, so need to fix